Uni_life_tracker/app/src/main/res/layout/activity_settings_page.xml

```diff
@@ -0,0 +1,109 @@
+            app:layout_constraintStart_toStartOf="parent"
+            android:layout_marginStart="16dp"
+            android:contentDescription="@string/left_arrow"/>
+
+    <TextView
+            android:layout_width="193dp"
+            android:layout_height="96dp"
+            android:text="@string/settings"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.072"
+            app:layout_constraintHorizontal_bias="0.366"
+            android:fontFamily="@font/roboto_mono_bold"
+            android:textSize="20sp"
+            android:textColor="#FF000000"
+            android:id="@+id/settings_title"/>
+    <TextView
+            android:layout_width="193dp"
+            android:layout_height="96dp"
+            android:text="@string/profile"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.234"
+            app:layout_constraintHorizontal_bias="0.146"
+            android:fontFamily="@font/roboto_mono_bold"
+            android:textSize="30sp"
+            android:textColor="#070707"
+            android:id="@+id/profile"/>
+
+
+    <TextView
+            android:layout_width="193dp"
+            android:layout_height="96dp"
+            android:text="@string/name"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.442"
+            app:layout_constraintHorizontal_bias="0.146"
+            android:textSize="20sp"
+            android:textColor="#070707"
+            android:id="@+id/profile_name"
+            android:fontFamily="@font/roboto_mono_light"/>
+    <TextView
+            android:layout_width="193dp"
+            android:layout_height="96dp"
+            android:text="@string/university"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.737"
+            app:layout_constraintHorizontal_bias="0.146"
+            android:textSize="20sp"
+            android:textColor="#070707"
+            android:id="@+id/profile_uni"
+            android:fontFamily="@font/roboto_mono_light"/>
+    <com.google.android.material.textfield.TextInputEditText
+            android:layout_width="170dp"
```

Comment:
I was wondering if the TextInputEditText could be next to the Name label rather than underneath it. This might have something to do with the type of Layout that was chosen at the top of the XML file

Reply:
maybe for each section you could enclose it in a LinearLayout & set orientation as horizontal?

Reply:
I can just drag it to the side